api/pkg/db/initializer/initializer.go

```diff
@@ -149,10 +148,10 @@ func addUsers(db *gorm.DB, log *log.Logger, data *app.Data) error {
 			}
 
 			// Add scopes for user if not added already
-			us := model.UserScope{UserID: user.ID, ScopeID: scope.ID}
-			q = db.Model(&model.UserScope{}).Where(&us)
+			us := &model.UserScope{UserID: user.ID, ScopeID: scope.ID}
```

Comment:
could you please revert this to the old usage? 
```go
q = db.Model(...).Where(&us)
```
makes it quite clear the `us` is passed as a `reference` and may be modified.